### PR TITLE
Add ordering for IP proxying, requirements document and TCP fallback

### PR DESCRIPTION
**Pull request by erickinnear**
Plus some other review comments. Separate commits as these are somewhat separate, but all touch the same text.

### CODE REVIEW

File: proposed_charter.md [hunk 1]
```diff
@@ -13,15 +13,23 @@ advantage of a unified congestion controller. An HTTP/3 datagram construct built
...
+Lastly, HTTP supports an established request/response semantic that can set up
 and configure flows for different services.
 
 The primary goal of this working group is to develop mechanisms that allow
 configuring and concurrently running multiple proxied stream- and datagram-based
-flows inside a HTTP/3 connection. The group will specify HTTP or HTTP/3
-extensions to enable this functionality within an HTTP/3 connection. The group
-will focus on a limited set of client-initiated services: (1) UDP CONNECT and
-(2) IP proxying.
+flows inside an HTTP connection. The group will specify HTTP and/or HTTP/3
```
Comment (DavidSchinazi) [inline proposed_charter.md:21]: I'm not sure this replacement of HTTP/3 with HTTP is correct, datagrams only exist in HTTP/3.

File: proposed_charter.md [hunk 2]
```diff
@@ -13,15 +13,23 @@ advantage of a unified congestion controller. An HTTP/3 datagram construct built
...
+extensions to enable this functionality. The group will focus on a limited set
+of client-initiated services: (1) UDP CONNECT and (2) IP proxying.
+
+The working group will first deliver a protcol solution for UDP CONNECT and a
+requirements document for IP proxying. Once complete, it will focus on a
```
Comment (DavidSchinazi) [inline proposed_charter.md]: nit: once both are complete

**Review by DavidSchinazi — APPROVED** [review]
Approved pending minor nits

File: proposed_charter.md (hunk 1 above)
Comment (erickinnear) [reply proposed_charter.md:21]: I kept the HTTP/3 specific bits for the motivation above, however isn't part of fallback saying that we'd want to include H2/1 in "configuring and concurrently running multiple proxied stream- and datagram-based flows"?

File: proposed_charter.md (hunk 2 above)
Comment (erickinnear) [reply proposed_charter.md]: Nice, updated in 50b1c6d

**Review by chris-wood — APPROVED** [review]
This matches latest discussions on the list. Thanks!

**Comment by martinduke** [comment]
LGTM

**Review by DavidSchinazi — APPROVED** [review]

**Pull request merged**